docs/aws.md

```diff
@@ -76,6 +76,13 @@ aws configure # Input your credentials here
 aws iam list-users
 ```
 
+Be sure to export AWS_ACCESS_KEY and AWS_SECRET_ACCESS_KEY so that kops can use them to make API calls
```

Comment:
I think it's good to call this out, but how about:

"If you don't want to use `aws configure` to store the credentials in your profile, then be sure to export...`